mod_authnz_pam.c

```diff
@@ -343,12 +345,7 @@ static void register_hooks(apr_pool_t * p) {
 #endif
 }
 
-#ifdef AP_DECLARE_MODULE
-AP_DECLARE_MODULE(authnz_pam)
-#else
-module AP_MODULE_DECLARE_DATA authnz_pam_module
-#endif
-	= {
+AP_DECLARE_MODULE(authnz_pam) = {
```

Comment:
It is true that by now we can assume Apache 2.4+ everywhere, so the `#ifdef` approach is no longer needed.

Reply:
I think the `AP_DECLARE_MODULE` even existed in 2.2.x. Not sure about 2.0.x though. But what I could gather from other modules, using both is perfectly valid and is done in most modules. Otherwise the `-n` flag is used in the Makefile to install the module via apxs.

Reply:
No, it got added in 2.3.6: https://nightlies.apache.org/httpd/trunk/doxygen/group__APACHE__CORE__CONFIG.html#ga662f6bc4b401d52b293a80b1768534af.